### ISSUE TITLE
auto migrations fails when table in schema

### ISSUE THREAD

**Issue by lherrman** [piccolo-orm/piccolo#883]
Hello,
I'm running the latest Postgres image, python 3.11.3 (venv) and piccolo 0.119.0.
When i create a new asgi application with a fresh venv, i can perfectly add, change and delete rows using the migrations new .. --auto / forward command. Then i can migrate the table into a schema e.g. `class Task(Table, schema="blog"):`, running forward will succefully put the table into that schema. But as soon as it is in a schema i'm not able to perform any auto migrations anymore. It won't find that table:
```
  - 2023-09-08T14:20:15:789180 [forwards]... The command failed.
relation "task" does not exist
Traceback (most recent call last):
  File "C:\venvs\piccolodb\Lib\site-packages\targ\__init__.py", line 448, in run
    command.call_with(arg_class)
  File "C:\venvs\piccolodb\Lib\site-packages\targ\__init__.py", line 229, in call_with
    asyncio.run(self.command(**cleaned_kwargs))
  File "C:\Program Files\Python311\Lib\asyncio\runners.py", line 190, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
  File "C:\Program Files\Python311\Lib\asyncio\runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Program Files\Python311\Lib\asyncio\base_events.py", line 653, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "C:\venvs\piccolodb\Lib\site-packages\piccolo\apps\migrations\commands\forwards.py", line 159, in forwards
    response = await run_forwards(
               ^^^^^^^^^^^^^^^^^^^
  File "C:\venvs\piccolodb\Lib\site-packages\piccolo\apps\migrations\commands\forwards.py", line 120, in run_forwards
    response = await manager.run()
               ^^^^^^^^^^^^^^^^^^^
  File "C:\venvs\piccolodb\Lib\site-packages\piccolo\apps\migrations\commands\forwards.py", line 97, in run
    return await self.run_migrations(app_config)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\venvs\piccolodb\Lib\site-packages\piccolo\apps\migrations\commands\forwards.py", line 82, in run_migrations
    await response.run()
  File "C:\venvs\piccolodb\Lib\site-packages\piccolo\apps\migrations\auto\migration_manager.py", line 863, in run
    await self._run_drop_columns(backwards=backwards)
  File "C:\venvs\piccolodb\Lib\site-packages\piccolo\apps\migrations\auto\migration_manager.py", line 642, in _run_drop_columns
    await self._run_query(
  File "C:\venvs\piccolodb\Lib\site-packages\piccolo\apps\migrations\auto\migration_manager.py", line 393, in _run_query
    await query.run()
  File "C:\venvs\piccolodb\Lib\site-packages\piccolo\query\base.py", line 445, in run
    return await engine.run_ddl(self.ddl[0], in_pool=in_pool)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\venvs\piccolodb\Lib\site-packages\piccolo\engine\postgres.py", line 553, in run_ddl
    response = await current_transaction.connection.fetch(ddl)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\venvs\piccolodb\Lib\site-packages\asyncpg\connection.py", line 620, in fetch
    return await self._execute(
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\venvs\piccolodb\Lib\site-packages\asyncpg\connection.py", line 1659, in _execute
    result, _ = await self.__execute(
                ^^^^^^^^^^^^^^^^^^^^^
  File "C:\venvs\piccolodb\Lib\site-packages\asyncpg\connection.py", line 1684, in __execute
    return await self._do_execute(
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\venvs\piccolodb\Lib\site-packages\asyncpg\connection.py", line 1731, in _do_execute
    result = await executor(stmt, None)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "asyncpg\protocol\protocol.pyx", line 201, in bind_execute
asyncpg.exceptions.UndefinedTableError: relation "task" does not exist
```

I can change the migration_file from
````
from piccolo.apps.migrations.auto.migration_manager import MigrationManager


ID = "2023-09-08T14:20:15:789180"
VERSION = "0.119.0"
DESCRIPTION = ""


async def forwards():
    manager = MigrationManager(
        migration_id=ID, app_name="home", description=DESCRIPTION
    )

    manager.drop_column(
        table_class_name="Task",
        tablename="task",
        column_name="completed_at",
        db_column_name="completed_at",
    )

    return manager

````

to 

````
....
    manager.drop_column(
        table_class_name="Task",
        tablename="blog.task",
        column_name="completed_at",
        db_column_name="completed_at",
    )

    return manager

````
it will perform the migration. I already spent a lot of time on it because i'm trying to setup a database at work. Any clues what's going on here?

**Comment by lherrman** [comment]
Ah oke. Good to know. Thank you very much for your quick reply.

**Comment by dantownsend** [comment]
This should be fixed in `piccolo==0.120.0`.

You might want to change any migrations you modified from this:

```python
    manager.drop_column(
        table_class_name="Task",
        tablename="blog.task",
        column_name="completed_at",
        db_column_name="completed_at",
    )
```

To this:

```python
    manager.drop_column(
        table_class_name="Task",
        tablename="task",  # <- NOTE
        column_name="completed_at",
        db_column_name="completed_at",
        schema="blog"  # <- NOTE
    )
```

Let me know if there are still any problems.
